test/Visualization_Tests/src/Helpers_Spec.enso

```diff
@@ -0,0 +1,35 @@
+from Standard.Base import all
+
+from Standard.Table.Data.Table import Table
+from Standard.Table.Data.Column import Column
+import Standard.Test
+import Standard.Visualization.Helpers
```

Comment:
Qualified imports should be above unqualified imports, separated by a line.

Reply:
Do we have any written coding guidelines for Enso code?